static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -306,6 +319,25 @@ function NativeFrame({
               </Tooltip>
             )}
           </GroupingCell>
+          {stacktraceChangesEnabled && hiddenFrameCount ? (
+            <ShowHideButton
```

Comment:
In a followup PR, it might be good to extract this out into a component that we can reuse in both stack traces.